tests_integration/test_package.py

```diff
@@ -0,0 +1,46 @@
+import logging
+import os
+import shutil
+from pathlib import Path
+from snowcli.cli.snowpark import package
+from unittest.mock import MagicMock, patch
+from tests_integration.snowflake_connector import snowflake_session, create_database
+from tests_integration.test_utils import contains_row_with, row_from_snowflake_session
+from tempfile import NamedTemporaryFile
+
+import pytest
+
+
+class TestPackage:
+    STAGE_NAME = "PACKAGE_TEST"
+
+    @pytest.mark.integration
+    def test_package_upload(self, runner, example_file, snowflake_session):
```

Comment:
Will integration tests of other `snowpark package` commands be added in later PRs?

Reply:
The other commands - lookup and create - do not connect to snowflake, they just . Therefore, their functionalities seem to be covered by unit tests. "Lookup" reaches to Anaconda - but it don't know if we want to call it in tests.